org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/view/task/TaskView.java

```diff
@@ -33,13 +39,7 @@
 import org.eclipse.ui.part.PageBook;
 import org.eclipse.ui.part.ViewPart;
 
-import org.eclipse.buildship.core.CorePlugin;
-import org.eclipse.buildship.core.configuration.ProjectConfiguration;
-import org.eclipse.buildship.ui.external.viewer.FilteredTree;
-import org.eclipse.buildship.ui.external.viewer.PatternFilter;
-import org.eclipse.buildship.ui.util.nodeselection.NodeSelection;
-import org.eclipse.buildship.ui.util.nodeselection.NodeSelectionProvider;
-import org.eclipse.buildship.ui.util.nodeselection.SelectionHistoryManager;
+import com.gradleware.tooling.toolingmodel.repository.FetchStrategy;
```

Comment:
Please use the the import order from the Buildship Oomph model. In case you don't use it, take the import order definition from [here](https://dl.dropboxusercontent.com/u/1159431/buildship.importorder)